<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteControlDialog</class>
 <widget class="QDialog" name="RemoteControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Remote Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="reflGroupBox">
       <property name="title">
        <string>Reflection Light</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QSpinBox" name="reflSpinBox">
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="reflOnPushButton">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="reflSetPushButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="reflOffPushButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="tranGroupBox">
       <property name="title">
        <string>Transmission Light</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QSpinBox" name="tranSpinBox">
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="tranSetPushButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="tranOnPushButton">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="tranOffPushButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="tranUvGroupBox">
       <property name="title">
        <string>UV Transmission Light</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QPushButton" name="tranUvOffPushButton">
          <property name="text">
           <string>Off</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="tranUvOnPushButton">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="tranUvSpinBox">
          <property name="maximum">
           <number>128</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="tranUvSetPushButton">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="sensorGroupBox">
     <property name="title">
      <string>Sensor Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="sensorStopPushButton">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sensorStartPushButton">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="gainLabel">
          <property name="text">
           <string>Gain:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="gainComboBox">
          <item>
           <property name="text">
            <string>Low</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Medium</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>High</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Maximum</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="intLabel">
          <property name="text">
           <string>Integration:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="intComboBox">
          <item>
           <property name="text">
            <string>100ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>200ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>300ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>400ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>500ms</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>600ms</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="modeLabel">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="modeComboBox">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Default</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VIS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UV</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="agcCheckBox">
        <property name="text">
         <string>Automatic Gain Control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reflReadPushButton">
        <property name="text">
         <string>Reflection Reading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tranReadPushButton">
        <property name="text">
         <string>Transmission Reading</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tranUvReadPushButton">
        <property name="text">
         <string>UV Transmisssion Reading</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sensor Readings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="ch0Label">
        <property name="text">
         <string>Channel 0:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="ch0LineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ch1Label">
        <property name="text">
         <string>Channel 1:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="ch1LineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>reflOffPushButton</tabstop>
  <tabstop>reflOnPushButton</tabstop>
  <tabstop>reflSpinBox</tabstop>
  <tabstop>reflSetPushButton</tabstop>
  <tabstop>tranOffPushButton</tabstop>
  <tabstop>tranOnPushButton</tabstop>
  <tabstop>tranSpinBox</tabstop>
  <tabstop>tranSetPushButton</tabstop>
  <tabstop>tranUvOffPushButton</tabstop>
  <tabstop>tranUvOnPushButton</tabstop>
  <tabstop>tranUvSpinBox</tabstop>
  <tabstop>tranUvSetPushButton</tabstop>
  <tabstop>sensorStopPushButton</tabstop>
  <tabstop>sensorStartPushButton</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>gainComboBox</tabstop>
  <tabstop>intComboBox</tabstop>
  <tabstop>agcCheckBox</tabstop>
  <tabstop>reflReadPushButton</tabstop>
  <tabstop>tranReadPushButton</tabstop>
  <tabstop>tranUvReadPushButton</tabstop>
  <tabstop>ch0LineEdit</tabstop>
  <tabstop>ch1LineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
